examples/00-mapdl-examples/lathe_cutter.py

```diff
@@ -0,0 +1,432 @@
+"""
+.. _ref_lathe_cutter_example:
+
+=====================================
+Structural Analysis of a Lathe Cutter
+=====================================
+
+**Summary**: Basic walk through demonstration of PyMAPDL capabilities.
+
+Objective
+=========
+
+The objective of this example is to highlight some of the often-used
+PyMAPDL categories via a lathe cutter finite element model. Lathe
+cutters have multiple avenues of wear and failure, and the analyses
+supporting their design would most often be transient
+thermal-structural. However, for simplicity, we will simulate this with
+a non-uniform load.
+
+.. figure:: ../../../_static/lathe_cutter_model.png
+    :align: center
+    :width: 600
+    :alt:  Lathe cutter geometry and load description.
+    :figclass: align-center
+
+    **Figure 1: Lathe cutter geometry and load description.**
+
+
+-  `OS <https://docs.python.org/3/library/os.html>`__ is used for Python
+   method to get the current working directory
+
+-  `Numpy <https://numpy.org/>`__ is used for Numpy arrays and functions
+
+Contents
+=========
+
+1. **Launch and variables:**
+   Define necessary variables and launch MAPDL.
+
+2. **Geometry, mesh and MAPDL parameters:**
+   Import geometry and inspect for MAPDL parameters. Define linear
+   elastic material model with Python variables. Mesh and apply symmetry
+   boundary conditions.
+
+3. **Coordinate system and load application:**
+   Create a local coordinate system for applied load and verify with
+   plot.
+
+4. **Pressure load application:**
+   Define the pressure load as a sine function of the length of the
+   application area using numpy arrays; import the pressure array into
+   MAPDL as a table array; verify the applied load, and solve.
+
+5. **Plotting:**
+   Show result plotting; plotting with selection; and working with the
+   plot legend.
+
+6. **Post processing:**
+   List a result two ways: PyMAPDL and Pythonic version of APDL.
+   Extended methods, and writing list to a file.
```

Comment:
```suggestion
6. **Postprocessing:**
   List a result two ways: use PyMAPDL and the Pythonic version of APDL.
   Demonstrate extended methods and writing a list to a file.
```

Reply:
I had to guess at how to change what was in line 60 to a sentence.

Reply:
This is good :)